.circleci/config.yml

```
@@ -109,6 +109,27 @@ commands:
                 WALLET_BALANCES_RECIPIENTS=sm://${GCP_PROJECT}/<< parameters.app_name >>_wallet_balances_recipients" \
             --image=${GCP_REGION}-docker.pkg.dev/${GCP_PROJECT}/${GCP_REGISTRY_NAME}/${DOCKER_IMAGE_NAME}:<< parameters.app_image_tag >>
 
+  cron-helm-deploy:
```

```suggestion
  deploy-cron-gke:
```

Essayer de trouver un nom sous la meme forme que `deploy-api-cloudrun`